### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Gok26** [projectdiscovery/naabu#542] [Q&A] [answered]
github.com/projectdiscovery/naabu/v2/cmd/naabu
/usr/local/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: cannot find -lpcap: No such file or directory
collect2: error: ld returned 1 exit status

how to fix this error?

**Comment by ehsandeep** [comment, marked as answer]
https://github.com/projectdiscovery/naabu#prerequisite
